index.html

```diff
@@ -811,25 +810,20 @@ <h3>The <dfn>toJSON</dfn> member</h3>
   <h3>The <dfn>PressureObserverOptions</dfn> dictionary</h3>
   <pre class="idl">
     dictionary PressureObserverOptions {
-      double sampleRate = 1.0;
+      [EnforceRange] unsigned long long sampleInterval = 0;
```

Comment:
I think an `unsigned long` is enough here. 4294967295 (its maximum value) corresponds to something like 1190 hours.

Reply:
It was mostly done for consistency with the brackground sync spec